android/java/org/chromium/chrome/browser/settings/NetworkListBaseAdapter.java

```diff
@@ -68,7 +68,7 @@
             mListener = listener;
         }
 
-        protected void updateNetworkInfo(EthereumChain item) {
+        protected void updateNetworkInfo(EthereumChain item, boolean activeNetwork) {
```

Comment:
nit: maybe protected can be removed, unless we wanna inherit from RowViewHolder.